wiki/fr/fr-Client-Troubleshooting.md

```diff
@@ -1,75 +1,77 @@
 
 ### Vous ne voyez pas le serveur que vous voulez rejoindre ?
 
-Vérifiez d'abord que vous avez le bon serveur de genre musical sélectionné dans votre fenêtre des paramètres de connexion. Mais il arrive parfois que des problèmes de réseau empêchent votre client d'afficher la liste de tous les serveurs disponibles. Si vous connaissez le nom du serveur que vous souhaitez rejoindre, vous pouvez [rechercher son adresse IP ici](http://explorer.jamulus.io/)). Saisissez l'adresse IP dans le champ « Adresse du serveur » de la fenêtre « Paramètre de connexion » pour vous y connecter. 
+Vérifiez d'abord que vous avez le bon serveur de genre musical sélectionné dans votre fenêtre des paramètres de connexion. Mais il arrive parfois que des problèmes de réseau empêchent votre client d'afficher la liste de tous les serveurs disponibles. Si vous connaissez le nom du serveur que vous souhaitez rejoindre, vous pouvez [rechercher son adresse IP ici](http://explorer.jamulus.io/)). Saisissez l'adresse IP dans le champ « Adresse du serveur » de la fenêtre « Paramètres de connexion » pour vous y connecter.
 
 ### Vous ne voyez pas du tout la liste des serveurs ?
 
-Au Royaume-Uni (et éventuellement dans d'autres régions/routeurs), le réglage du modem Internet par câble de Virgin Media peut poser problème. La case « Block Fragmented IP Packets » ne doit pas être cochée. Pour les autres routeurs/FAI, essayez également de désactiver le SPI (Stateful Packet Inspection) et si cela résout le problème, vous pouvez décider de le laisser désactivé ou non.
+Au Royaume-Uni (et éventuellement dans d'autres régions/routeurs), le réglage du modem Internet par câble de Virgin Media peut poser problème. « Block Fragmented IP Packets » ne doit pas être cochée. Pour les autres routeurs/FAI (fournisseur d'accès internet), essayez également de désactiver le SPI (Stateful Packet Inspection) et si cela résout le problème, vous pouvez décider de le laisser désactivé ou non.
 
-Dans certains cas, il se peut que ce soit votre fournisseur d'accès Internet qui bloque votre utilisation pour Jamulus. Voir la note sur la [page de dépannage du serveur](Server-Troubleshooting#personne-ne-peut-se-connecter-à-mon-serveur-mais-je-peux-my-connecter-localement) à propos de « Personne ne peut se connecter à mon serveur ».
+Dans certains cas, il se peut que ce soit votre FAI qui bloque votre utilisation pour Jamulus. Voir la note sur la [page de dépannage du serveur](Server-Troubleshooting#personne-ne-peut-se-connecter-à-mon-serveur-mais-je-peux-my-connecter-localement) à propos de « Personne ne peut se connecter à mon serveur ».
 
-### Des problèmes pour utiliser Garageband (ou un autre logiciel de MAO) avec Jamulus ? 
+### Des problèmes pour utiliser Garageband (ou autre STAN) avec Jamulus ? 
```

Comment:
I suggest `<abbr title="Station de travail audio numérique">STAN</abbr>` here.

Reply:
As usual, not before the English "DAW" got a abbr too. Let's discuss that for after the 3.6.2 release.